engine/src/java/com/runrev/android/nativecontrol/InputControl.java

```diff
@@ -247,7 +247,7 @@ public void setIsPassword(boolean p_password)
         {
             m_transformation_method = m_text_view.getTransformationMethod();
             m_text_view.setTransformationMethod(PasswordTransformationMethod.getInstance());
-            m_text_view.setInputType(InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS);
+            m_text_view.setInputType(InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS | InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD);
```

Comment:
Why use TYPE_TEXT_VARIATION_VISIBLE_PASSWORD instead of TYPE_TEXT_VARIATION_PASSWORD here?
Don't we want the contents to be hidden from the user?

Reply:
@livecodeian PR Updated :)